app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    get_command = command.split(" ")
+    if get_command[0] == "mv" and len(get_command) == 3:
+        old_file_name = get_command[1]
+        new_file_name = get_command[2]
+        if "/" in new_file_name:
```

Comment:
Do not track specific directory delimiters in your code that are specific to the platform. Use built-in methods in `os` to make your code platform-agnostic.